addons/mail/static/src/core/common/mail_core_common_service.js

```diff
@@ -32,24 +32,6 @@ export class MailCoreCommon {
                 message.delete();
             }
         });
-        this.busService.subscribe("mail.message/toggle_star", (payload, { id: notifId }) => {
-            const { message_ids: messageIds, starred } = payload;
-            for (const messageId of messageIds) {
-                const message = this.store.Message.insert({ id: messageId, starred });
```

Comment:
The insert part of the toggle star should say in common (assuming star on message is a common feature, which I see no reason not to be).